north/src/manifest.rs

```diff
@@ -140,22 +142,31 @@ pub struct Manifest {
     /// Target arch
     pub arch: String,
     /// Path to init
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
why is there any case where we want to skip serialization of the manifest @inorick ?

Reply:
For yaml a option can be either not preset or set to `null`. Semantically this is the same. Skipping `None`s during serialisation reduces the size of the manifest. In the examples we also do not have `None`s.